src/transformers/feature_extraction_utils.py

```diff
@@ -210,10 +210,10 @@ def to(self, *args, **kwargs) -> "BatchFeature":
         # We cast only floating point tensors to avoid issues with tokenizers casting `LongTensor` to `FloatTensor`
         for k, v in self.items():
             # check if v is a floating point
-            if torch.is_floating_point(v):
+            if torch.is_tensor(v) and torch.is_floating_point(v):
                 # cast and send to device
                 new_data[k] = v.to(*args, **kwargs)
-            elif device is not None:
+            elif torch.is_tensor(v) and device is not None:
```

Comment:
I had to add those checks since in this PR we return some elements as raw lists / tuples. Otherwise the current slow tests fail because `processor(inputs).to(device)` will be called on tuples and lists 

cc @sgugger @amyeroberts

Reply:
False alarm, reverted that commit!